### DISCUSSION TITLE
How to custom theme?

### DISCUSSION THREAD

**Discussion by SnowingFox** [onu-ui/onu-ui#130] [Q&A]
In onu-ui, is there a way to custom theme?

**Comment by zyyv** [comment]
> In onu-ui, is there a way to custom theme?

No way, it’s not perfect yet, don’t need to understand it for the time being.😅

**Comment by yzh990918** [comment]
We'll present the scheme later, skinning is not currently supported, but you can look at the 'colors' folder under' packages/preset ', all color schemes in the component library are implemented based on this set of variables.

**Comment by SebastienPingal** [comment]
Don't want to push you there but I love Onu and would love to be able to customize my themes !

**Reply by SebastienPingal** [reply]
btw, I'm a newbie dev but if I can be of any help, I'd love to save some time to help you guys.

**Reply by zyyv** [reply]
Thank you for your kindness, but we are not very active in updating and maintaining, and our attitude may affect your enthusiasm. Sorry, it's like a toy project now.
